angular.json

```diff
@@ -223,6 +386,20 @@
             }
           }
         },
+        "serve-ssr-cds": {
+          "builder": "@nguniversal/builders:ssr-dev-server",
+          "options": {
+            "browserTarget": "storefrontapp:build:cds.development",
+            "serverTarget": "storefrontapp:server-cds"
```

Comment:
In this case we also need fileReplacement for "default" mode of server-cds, not only in production.

Reply:
I thought you would only set here: "serverTarget": "storefrontapp:server-cds:cds.development" without adding extra configuration here.

Reply:
Seemed safer this way. Do you agree?

Reply:
I would avoid completely the `cds.development` configuration here and use set it in this `401` line. It was already missed in package.json. ;)